com.unity.ml-agents/Documentation~/test-package-guide.md

```diff
@@ -0,0 +1,38 @@
+# About the &lt;package name&gt; tests
```

Comment:
Should we just delete these placeholder files? Or at least make a note of which ones we need to go update later...

Reply:
I'll delete a bunch of these.

Reply:
done